sdk/schemaregistry/schema-registry/recordings/node/schemaregistryclient/recording_gets_schema_id.json

```diff
@@ -0,0 +1,93 @@
+{
```

Comment:
These are new json browser recordings so I believe you should delete the now obselete .js ones.

Reply:
good point